main/webapp/modules/core/scripts/views/data-table/column-header-ui.js

```diff
@@ -72,29 +72,31 @@ DataTableColumnHeaderUI.prototype._render = function() {
   });
 
   var serviceUrl = null;
-  var service = null;
-  var serviceLogo = null;
   if (this._column.reconConfig) {
    serviceUrl = this._column.reconConfig.service;
   }
-  try {
-    if (new URL(serviceUrl)) {
-      service = ReconciliationManager.getServiceFromUrl(serviceUrl);
-    }
-    if (service) {
-      serviceLogo=service.logo;
-   }
-
-    var img = $("<img>");
-    if (serviceLogo) {
-      var imageUrl = serviceLogo;
-      img.attr("src", imageUrl);
-      img.attr("title", service.name);
-      img.addClass("serviceLogo")
-      img.appendTo(elmts.serviceLogoContainer.show());
+  if (serviceUrl) {
+    try {
+      var service = null;
+      var serviceLogo = null;
+      if (new URL(serviceUrl)) {
+        service = ReconciliationManager.getServiceFromUrl(serviceUrl);
+      }
+      if (service) {
+        serviceLogo = service.logo;
```

Comment:
Shouldn't there be a `new URL()` check here just as in `schema-alignment.js`?

Reply:
Sure, I've added it underneath as at this stage `serviceLogo` can still be `undefined`.